src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/spreadsheet/outbound/PricesSpreadsheetGeneratorTest.kt

```diff
@@ -102,19 +104,31 @@ internal class PricesSpreadsheetGeneratorTest {
   @Test
   internal fun `check tabs`() {
     val sheetFile = pricesSpreadsheetGenerator.generate(Supplier.GEOAMEY, timeSource.date())
-    val sheet = XSSFWorkbook(FileInputStream(sheetFile))
+    val workbook = XSSFWorkbook(FileInputStream(sheetFile))
```

Comment:
Extending this test to check the move reference is as expected.  For this I am actually using the move type as a move reference key.